tests/eigenwave3d.py

```diff
@@ -246,6 +252,8 @@ def main():
                        formatter_class=RawTextHelpFormatter)
     p.add_argument('mode', choices=('default', 'read', 'converge', 'cx1'),
                    nargs='?', default='default', help=ModeHelp)
+    p.add_argument('-so', '--spatial_order', default=2, type=int, dest='so',
+                   help='Accuracy order to use for code generation, eg. 2 for 4th order in x,y,z')
```

Comment:
"Accuracy order" is confusing. From the name of the argument, you mean the order of the spatial discretisation. The fourth order convergence that you see is the (expected) order of the solution's convergence. You should make this distinction.